src/App.js

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useRef, useState } from "react";
+import download from "downloadjs";
+import { toPng } from "html-to-image";
+import Draggable from "react-draggable";
+import logo from "./logo.svg";
+
+const URL = "https://api.imgflip.com/get_memes";
+
+export default function App() {
```

Comment:
I have seen that in your project you have used the arrow function, but here you didn't use it, for more readability you can convert it into the arrow function.